soh/src/code/z_skelanime.c

```diff
@@ -478,6 +478,11 @@ void SkelAnime_GetFrameData(AnimationHeader* animation, s32 frame, s32 limbCount
             LOG_ADDRESS("tbl", staticData);
         }
 
+        lusprintf(__FILE__, __LINE__, 2, "SkelAnime_GetFrameData: jointIndices->z >= staticIndexMax: %i", jointIndices->z >= staticIndexMax);
```

Comment:
I assume the enums aren't available here in C?

Reply:
yeah, enums weren't available so i'm just magic numbering it for now